platform.md

```diff
@@ -89,6 +92,18 @@ Where:
 - `-group` MUST specify input from a `group.toml` file path as defined in the [Data Format](#data-format) section.
 - `-plan` MUST specify input from a Build Plan as defined in the [Buildpack Interface Specification](buildpack.md).
 
+#### Export phase
```

Comment:
Adding the Export phase and it's usage here without also adding the Analyze and Restore phases to the spec seems more confusing to me than omitting all three.

Reply:
Isn't that a bit out of scope of their proposed change, though?

Reply:
Ah I see, (at least I think I'm understanding): you're suggesting we don't add it to the spec proper, but instead include the feature in the lifecycle, but leave the exporter out of the spec until we add all three missing phases and generally overhaul the lifecycle in the spec?

Reply:
I am happy to own updating the platform spec, it's long overdue. Afterward we can accept incremental changes. 

Given that this is a specification rather than documentation adding Exporter with just this one flag is more confusing than not having it at all. By choosing to document a single flag we incorrectly imply something about its importance.

Reply:
Is `CNB_PLATFORM_DIR` part of the spec though if we're reserving that keyword?